config.mk

```diff
@@ -78,6 +78,12 @@ export FPC_PATH=$(abspath $(TOP))
 # re-defined to point to the FPC path as seen by the docker daemon
 export DOCKERD_FPC_PATH ?= $(FPC_PATH)
 
+# For testing Fabric binaries are needed; you can customize these via the following
+# env variable. By default we fetch the binaries into $(FPC_PATH)/fabric/_internal/bin
+export FABRIC_BIN_DIR ?= $(FPC_PATH)/fabric/_internal/bin
+# In case you want to your custom fabric bins, just point to your fabric path
+# export FABRIC_BIN_DIR ?= $(FABRIC_PATH)/build/bin
```

Comment:
Leftover?

Reply:
this was on purpose but I rephrased it now